build/repo.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+   <ItemGroup>
+     <ExcludeFromPack Include="$(RepositoryRoot)src\Microsoft.AspNetCore.AzureAppServices.TestBundle\Microsoft.AspNetCore.AzureAppServices.TestBundle.csproj" />
```

Comment:
Should we leave the TestBundle in the src directory? Do we expect to publish it for every build of this repo or does it have a different lifecycle and gets pushed by invoking the `PushSiteExtension` target somewhere in our builds? If the latter is true, let's move the project out of src/